examples/cloud-router-2-aws-connection/terraform.tfvars.example

```diff
@@ -1,5 +1,6 @@
 equinix_client_id      = "<MyEquinixClientId>"
 equinix_client_secret  = "<MyEquinixSecret>"
+equinix_endpoint       = "endpoint_URL"
```

Comment:
Don't need the endpoint_url for the customer in the examples as noted in other comments.